src/api/entities/Identity/index.ts

```diff
@@ -279,9 +283,19 @@ export class Identity extends Entity<UniqueIdentifiers, string> {
       context,
     } = this;
 
-    const assembleResult = ({ primaryKey }: PolymeshPrimitivesIdentity): PermissionedAccount => {
+    const assembleResult = ({
+      primaryKey,
+    }: PolymeshPrimitivesIdentityDidRecord): PermissionedAccount => {
+      if (primaryKey.isNone) {
+        throw new PolymeshError({
```

Comment:
I left a [question](https://polymathnetwork.slack.com/archives/C01RLA867C5/p1653668029041419) in the mesh questions channel because it seems weird to me that this value is an `Option`